### PR TITLE
Suppress module name spew when running tests.

### PR DESCRIPTION
**Pull request by tseaver**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -6,15 +6,15 @@ envlist =
 deps =
     pytest
 covercmd =
-    py.test \
+    py.test -q \
```
Comment (dhermes) [inline tox.ini]: Can you use `--quiet`

File: tox.ini [hunk 2]
```diff
@@ -6,15 +6,15 @@ envlist =
...
       --cov=google.cloud \
       --cov=unit_tests \
       --cov-config {toxinidir}/.coveragerc \
       unit_tests
 
 [testenv]
 commands =
-    py.test {posargs} unit_tests
+    py.test -q {posargs} unit_tests
```
Comment (dhermes) [inline tox.ini]: Can you use `--quiet`

**Comment by dhermes** [comment]
@tseaver I'm unclear what spew is being quited here. Care to give a bit more detail?

**Comment by tseaver** [comment]
@dhermes py.test defaults to verbosity == 1, which injects the unit test module name into the output., e.g.:

```
unit_tests/test__helpers.py .....................................................................
unit_tests/test_client.py ............
unit_tests/test_connection.py ...........................
unit_tests/test_credentials.py .............
unit_tests/test_exceptions.py .....
unit_tests/test_iterator.py ..................
unit_tests/test_operation.py ..............
unit_tests/bigquery/test__helpers.py ............................................
unit_tests/bigquery/test_client.py ...............
unit_tests/bigquery/test_connection.py ..
unit_tests/bigquery/test_dataset.py .............................................
unit_tests/bigquery/test_job.py ......................................................
unit_tests/bigquery/test_query.py ...............
unit_tests/bigquery/test_schema.py ...........
unit_tests/bigquery/test_table.py ....................................................................
unit_tests/bigtable/test_client.py .............................
unit_tests/bigtable/test_cluster.py ...................
unit_tests/bigtable/test_column_family.py ...........................................
unit_tests/bigtable/test_instance.py .........................
unit_tests/bigtable/test_row.py ....................................
unit_tests/bigtable/test_row_data.py ...................................................................................
unit_tests/bigtable/test_row_filters.py ...............................................................................
unit_tests/bigtable/test_table.py ..............................
unit_tests/datastore/test_batch.py ......................
unit_tests/datastore/test_client.py ................................................
unit_tests/datastore/test_connection.py .............................................
unit_tests/datastore/test_entity.py ...............
unit_tests/datastore/test_helpers.py .......................................................................
unit_tests/datastore/test_key.py .................................................
unit_tests/datastore/test_query.py ........................................................
unit_tests/datastore/test_transaction.py ..........
unit_tests/dns/test_changes.py ................
unit_tests/dns/test_client.py ........
unit_tests/dns/test_connection.py ..
unit_tests/dns/test_resource_record_set.py ......
unit_tests/dns/test_zone.py ..........................
unit_tests/error_reporting/test_client.py .....
unit_tests/language/test_client.py .......
unit_tests/language/test_connection.py .
unit_tests/language/test_document.py .............
unit_tests/language/test_entity.py ..
unit_tests/language/test_sentiment.py ..
unit_tests/language/test_syntax.py .....
unit_tests/logging/test__gax.py ...............................................
unit_tests/logging/test_client.py ...................
unit_tests/logging/test_connection.py ............................
unit_tests/logging/test_entries.py ........
unit_tests/logging/test_logger.py .............................
unit_tests/logging/test_metric.py ..............
unit_tests/logging/test_sink.py ..............
unit_tests/logging/handlers/test_handlers.py ....
unit_tests/logging/handlers/transports/test_background_thread.py ......
unit_tests/logging/handlers/transports/test_base.py .
unit_tests/logging/handlers/transports/test_sync.py ..
unit_tests/monitoring/test__dataframe.py ssssssssss...
unit_tests/monitoring/test_client.py ................
unit_tests/monitoring/test_connection.py .
unit_tests/monitoring/test_group.py ............................
unit_tests/monitoring/test_label.py .........
unit_tests/monitoring/test_metric.py .....................
unit_tests/monitoring/test_query.py .................................
unit_tests/monitoring/test_resource.py ...........
unit_tests/monitoring/test_timeseries.py ...........
unit_tests/pubsub/test__gax.py ...............................................
unit_tests/pubsub/test__helpers.py ....
unit_tests/pubsub/test_client.py ............
unit_tests/pubsub/test_connection.py .......................................
unit_tests/pubsub/test_iam.py ..............
unit_tests/pubsub/test_message.py ........
unit_tests/pubsub/test_subscription.py ........................................
unit_tests/pubsub/test_topic.py ....................................
unit_tests/resource_manager/test_client.py ..........
unit_tests/resource_manager/test_connection.py ..
unit_tests/resource_manager/test_project.py .................
unit_tests/storage/test__helpers.py ..........
unit_tests/storage/test_acl.py ...................................................................
unit_tests/storage/test_batch.py ......................
unit_tests/storage/test_blob.py ....................................................................................
unit_tests/storage/test_bucket.py .......................................................................
unit_tests/storage/test_client.py ....................
unit_tests/storage/test_connection.py ..
unit_tests/streaming/test_buffered_stream.py ........
unit_tests/streaming/test_exceptions.py ....
unit_tests/streaming/test_http_wrapper.py .............................
unit_tests/streaming/test_stream_slice.py ......
unit_tests/streaming/test_transfer.py ....................................................................................................................................
unit_tests/streaming/test_util.py ......
unit_tests/translate/test_client.py .............
unit_tests/translate/test_connection.py ..
unit_tests/vision/test_client.py ...............
unit_tests/vision/test_connection.py .
unit_tests/vision/test_entity.py .
unit_tests/vision/test_face.py ......
unit_tests/vision/test_feature.py ...
unit_tests/vision/test_image.py ...
```
With the PR, it is just dots (less noise, more signal):

```
........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................ssssssssss...................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
```

File: tox.ini (hunk 1 above)
Comment (tseaver) [inline tox.ini]: 00798a5

File: tox.ini (hunk 2 above)
Comment (tseaver) [inline tox.ini]: 00798a5

**Comment by dhermes** [comment]
LGTM

**Pull request merged**